redis-cluster/build-dropins.sh

```
@@ -0,0 +1,66 @@
+if [[ ! -d "$IDEMPIERE_SRC/org.idempiere.redis.service" ]]; then
+  echo "ERROR: $IDEMPIERE_SRC does not contain org.idempiere.redis.service/" >&2
+  echo "Make sure IDEMPIERE_SRC points at the iDempiere fork that has the bundle." >&2
+  exit 2
+fi
+
+DOCKER_DIR="$(cd "$(dirname "$0")" && pwd)"
+MANIFEST="$IDEMPIERE_SRC/org.idempiere.redis.service/META-INF/MANIFEST.MF"
+TARGET_BASE_VERSION="13.0.0"
+
+if [[ "$(uname -s)" == "Darwin" ]] && [[ -z "${JAVA_HOME:-}" ]]; then
+  export JAVA_HOME="$(/usr/libexec/java_home -v 17)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)